src/server/game/Spells/Spell.cpp

```diff
@@ -2448,7 +2448,7 @@ void Spell::DoAllEffectOnTarget(TargetInfo* target)
         // Needs to be called after dealing damage/healing to not remove breaking on damage auras
         DoTriggersOnSpellHit(spellHitTarget, mask);
 
-        // if target is fallged for pvp also flag caster if a player
+        // if target is flagged for pvp also flag caster if a player
         if (unit->IsPvP() && m_caster->GetTypeId() == TYPEID_PLAYER)
             m_caster->ToPlayer()->UpdatePvP(true);
```

Comment:
I suggest to use Step Into/Step Over debugger features, http://www.codeproject.com/KB/cs/MasteringInDebugging.aspx has some nice tips on How To Replace Printf Debugging With Proper Debugging Tools :)